.taskcluster.yml

```diff
@@ -216,15 +223,25 @@ tasks:
         priority: medium
         retries: 5
         scopes:
-          - queue:scheduler-id:${scheduler_id}
-          - queue:create-task:highest:aws-provisioner-v1/gecko-focus
-          - queue:create-task:highest:scriptworker-prov-v1/mobile-signing-v1
-          - queue:create-task:highest:scriptworker-prov-v1/mobile-pushapk-v1
-          - project:mobile:focus:releng:signing:cert:release-signing
-          - project:mobile:focus:releng:signing:format:autograph_focus
-          - project:mobile:focus:releng:googleplay:product:focus
-          - secrets:get:project/focus/tokens
-          - queue:route:index.project.mobile.focus.nightly.latest
+          $flatten:
+            - queue:scheduler-id:${scheduler_id}
+            - queue:create-task:highest:aws-provisioner-v1/gecko-focus
```

Comment:
I think this is defunct now. Or it's about to be. Might be a good moment to update this too.